python/ray/serve/pipeline/deployment_node.py

```diff
@@ -61,6 +60,7 @@ def replace_with_handle(node):
                 )
             elif isinstance(node, (DeploymentMethodNode, DeploymentFunctionNode)):
                 from ray.serve.pipeline.json_serde import DAGNodeEncoder
+                from ray.serve.deployment_graph import RayServeDAGHandle
```

Comment:
why move this?

Reply:
if this is due to a new circular dependency, let's try to fix it directly instead of changing this?

Reply:
sure let me give a try